clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -19489,6 +19489,12 @@ Value *CodeGenFunction::EmitAMDGPUBuiltinExpr(unsigned BuiltinID,
         F, {EmitScalarExpr(E->getArg(0)), EmitScalarExpr(E->getArg(1)),
             EmitScalarExpr(E->getArg(2)), EmitScalarExpr(E->getArg(3))});
   }
+  case AMDGPU::BI__builtin_amdgcn_s_prefetch_data: {
+    llvm::Value *Ptr = EmitScalarExpr(E->getArg(0));
```

Comment:
can use `emitBuiltinWithOneOverloadedType<1>(...)`

Reply:
I does not have a return value, so cannot.

Reply:
`emitBuiltinWithOneOverloadedType` doesn't have to have a return value

Reply:
Hm, ok, The description of the function is misleading. Done.